scripts/docs.js

```diff
@@ -20,8 +22,14 @@ function validatePublicFiles(definitionFiles) {
   }
 }
 
+function getNodeModulesInterfacePath(componentName) {
+  return cwd + "/node_modules/@cloudscape-design/components/" + componentName + "/interfaces.d.ts";
```

Comment:
Is there any reason that we haven't used `path.resolve(...)`? 🤔 
This would use current working directory by default as a base url so there will be no need for line `9`

Ref: https://nodejs.org/api/path.html#pathresolvepaths

Reply:
Not really, I simply used cwd because the other scripts were also using it so I don't mind changing it to path.resolve, it may make more sense here.

Reply:
There's not a big difference between the two IMO, both are going to concatenate path segments anyways, but `path.resolve` offers a bit of better protection against accidental `/` overlaps and etc.